src/gui/accountsettings.cpp

```diff
@@ -267,9 +269,10 @@ void AccountSettings::slotEncryptFolderFinished(int status)
         QMessageBox::warning(nullptr, tr("Warning"), job->errorString());
     }
 
-    const auto folderInfo = job->property(propertyFolderInfo).value<FolderStatusModel::SubFolderInfo*>();
-    Q_ASSERT(folderInfo);
-    const auto index = _model->indexForPath(folderInfo->_folder, folderInfo->_path);
+    const auto folder = job->property(propertyFolder).value<Folder *>();
+    const auto path = job->property(propertyPath).value<QString>();
+    Q_ASSERT(folder);
+    const auto index = _model->indexForPath(folder, path);
     Q_ASSERT(index.isValid());
```

Comment:
@FlexW Is error handling needed here? Or, maybe, just a warning log?

Reply:
I'll add a warning